app/src/components/PriceLevel.tsx

```diff
@@ -1,22 +1,22 @@
 import React, { Component } from 'react';
 import { withStyles, WithStyles, createStyles } from '@material-ui/core/styles';
 import classNames from 'classnames';
+import { CSSTransition, TransitionGroup } from 'react-transition-group';
 
 import { Box } from '@material-ui/core';
 import { Styles } from '../styles/PriceLevel';
 import Order from './Order';
-
 import { ordersEquals } from '../utils/order-book-utils';
 import { roundNumber } from '../utils/number-utils';
 import { TransactionType, Order as OrderType } from '../models/OrderBook';
 
 const styles = createStyles(Styles);
 
-interface Props extends WithStyles<typeof styles>{
-    orders: Array<OrderType>,
-    maxQuantity: number,
-    type: TransactionType,
-    price: number,
+interface Props extends WithStyles<typeof styles> {
+    orders: Array<OrderType>;
+    maxQuantity: number;
+    type: TransactionType;
+    price: number;
```

Comment:
if you're gonna do linting changes, can you add the rules to the eslintrc

Reply:
Ah didn't see those changes I'll revert it